apps/server/src/modules/fwu-learning-contents/fwu-learning-contents.module.ts

```diff
@@ -6,14 +6,15 @@ import { AuthorizationModule } from '@modules/authorization';
 import { HttpModule } from '@nestjs/axios';
 import { Module, NotFoundException } from '@nestjs/common';
 import { ConfigModule } from '@nestjs/config';
-import { Account, Role, SchoolEntity, SchoolYearEntity, SystemEntity, User } from '@shared/domain/entity';
+import { Role, SchoolEntity, SchoolYearEntity, SystemEntity, User } from '@shared/domain/entity';
```

Comment:
too deep import, dont go past module index file

Reply:
@shared/domain does not export the entities